Musicca/app/src/main/java/com/example/musicca/fragments/PeopleFragment.java

```diff
@@ -0,0 +1,105 @@
+package com.example.musicca.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.example.musicca.R;
+import com.example.musicca.adapters.PeopleAdapter;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PeopleFragment extends Fragment {
+
+    public static final String TAG = "PeopleFragment";
+
+    private RecyclerView rvPeople;
+    protected PeopleAdapter peopleAdapter;
+    protected List<ParseUser> allUsers;
+    private SwipeRefreshLayout swipeContainer;
+
+    public PeopleFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_people, container, false);
+
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        Toast.makeText(getContext(), "View Created!", Toast.LENGTH_SHORT).show();
+
+        // using recycler view in a fragment, create a view for on row in the list
+        rvPeople = view.findViewById(R.id.rvPeople);
+        allUsers = new ArrayList<>();
+
+        queryUsers(0);
+
+        // Lookup the swipe container view
+        swipeContainer = view.findViewById(R.id.swipeContainer);
+        // Configure the refreshing colors
+        swipeContainer.setColorSchemeResources(android.R.color.holo_blue_bright,
+                android.R.color.holo_green_light,
+                android.R.color.holo_orange_light,
+                android.R.color.holo_red_light);
+        swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                // Remember to CLEAR OUT old items before appending in the new ones
+                peopleAdapter.clear();
+                queryUsers(0);
+                peopleAdapter.notifyDataSetChanged();
+            }
+        });
+    }
+
+    protected void queryUsers(int page) {
```

Comment:
What is `page` supposed to do? It's not used. 
Also, on this fragment, do you show all the users in the database?

Reply:
Yes, I show all the users in the database in this fragment. 
I also got rid of the int page, I realized I was no longer using it